ciphey/basemods/Checkers/ezcheck.py

```diff
@@ -45,6 +46,8 @@ def __init__(self, config: Config):
         # First PyWhat, as it's the fastest
         self.checkers.append(config(What))
 
+        self.checkers.append(config(Quadgrams))
+
```

Comment:
This should be below the json checker, no? Actually, can we get rid of the json checker with pywhat and quadgrams or?

Reply:
You're right it should be below the json checker since it's ranked by speed. I'll change that. Also no we can't get rid of the json checker. You can't identify json with regex and the quadgrams checker won't catch json if there's not that many words in it. The way we're doing it currently is really the only way by trying to load the output as json and seeing if it fails or not.

Reply:
No, I'm saying the content _in_ json would be caught by pywhat / the quadgrams, right? It's very very unlikely to see JSON like:

```json
{"JOP JP@J( RJWJPIWEJP JOI@E JPWE PJIWEQJPIO WPJOFSPJOSFPJ" : "KOPDJOPASDJ( ASJ(J(EQ@ J(DJOJOSJIOADSIOFIOFSOIJ}
```

Instead, it'll look like:

```json
  "main":{
     "temp":306.15, //current temperature
     "pressure":1013,
     "humidity":44,
     "temp_min":306, //min current temperature in the city
     "temp_max":306 //max current temperature in the city
   }
```

Which, would get caught by PyWhat at 0.0 rarity or quadgrams?I just think we don't need a JSON checker, why JSON and not XML? Or YAML? Or TOML? And if the JSON contains English words and values that match PyWhat -- why keep it? 😄

Reply:
I guess you're right since the example from this issue (https://github.com/Ciphey/Ciphey/issues/183) gets caught by the quadgrams checker. But sometimes there won't be enough words for the quadgrams checker to work. Also what if the json properties are encoded/encrypted themselves? Only the json checker would work in that case. We originally added the json checker because someone opened an issue where Ciphey didn't catch json. Yes we should also add XML, YAML, etc. checkers too.